Assets/HoloToolkit-Examples/Input/Scripts/TestOverrideFocusedObject.cs

```diff
@@ -10,6 +10,7 @@ public class TestOverrideFocusedObject : MonoBehaviour, IInputClickHandler
         private InputManager inputManager;
         private TextMesh textMesh;
 
+        //Use this for initialisation
```

Comment:
This doesn't add anything to what isn't already visible in code. Let's remove this one.